packages/muelu/src/Utils/MueLu_Utilities_kokkos_def.hpp

```diff
@@ -103,8 +104,23 @@
 
 #include <MueLu_Utilities_kokkos_decl.hpp>
 
+#include <KokkosKernels_Handle.hpp>
+#include <KokkosSparse_partitioning_impl.hpp>
+
+
 namespace MueLu {
 
+  template<class ViewType, class execution_space = typename ViewType::execution_space>
```

Comment:
Oh I missed this. We could just delete this now, and also you're swapping ``i`` and ``N-2``, not ``N-1-i``.

Reply:
Will delete on next PR (when I integrate this goop into the Pairwise aggregation.